docs/additional-features/chainbridge/overview.md

```diff
@@ -3,42 +3,28 @@ id: overview
 title: Overview
 ---
 
-ChainBridge is a modular multi-directional blockchain bridge supporting EVM and Substrate compatible chains. It allows users to transfer whatever kind of assets or messages to another chain.
+## What is ChainBridge?
 
-To find out more about ChainBridge, visit the [official docs](https://chainbridge.chainsafe.io/).
+ChainBridge is a modular multi-directional blockchain bridge supporting EVM and Substrate compatible chains, built by ChainSafe. It allows users to transfer all kinds of assets or messages between two different chains.
 
-## Architecture
+To find out more about ChainBridge, please first visit the [official docs](https://chainbridge.chainsafe.io/) provided by its developer.
 
-In ChainBridge, there are three types of contracts called Bridge/Handler/Target in each chain and relayers to communicate between chains.
+This guide is intended to help with the Chainbridge integration to Polygon Edge. It walks through the setup of a bridge between a running Polygon PoS (Mumbai testnet) and a local Polygon Edge network. 
 
-### Bridge contract
+## Requirements
 
-A Bridge contract that manages requests, votes, executions needs to be deployed in each chain. Users will call `deposit` in Bridge to start a transfer and Bridge delegates the process to Handler contract corresponding to Target contract. Once Handler contract has been successful in call Target contract, Bridge contract emits `Deposit` event to notify relayers.
+In this guide, you will run Polygon Edge nodes, a ChainBridge relayer (more about it  [here](https://edge-docs.polygon.technology/docs/additional-features/chainbridge/definitions) ), and the cb-sol-cli tool, which is a CLI tool to deploy contracts locally (you can check [this](https://chainbridge.chainsafe.io/cli-options/#cli-options) too). The following environments are required before starting the setup:
```

Comment:
Please don't attach protocol and domain for internal link. This link will be broken when changing domain. You can just set the path for internal link

```suggestion
In this guide, you will run Polygon Edge nodes, a ChainBridge relayer (more about it  [here](/docs/additional-features/chainbridge/definitions) ), and the cb-sol-cli tool, which is a CLI tool to deploy contracts locally (you can check [this](https://chainbridge.chainsafe.io/cli-options/#cli-options) too). The following environments are required before starting the setup:
```